kickoff/test/views/test_json.py

```diff
@@ -236,7 +236,9 @@ def testThunderbirdVersions(self):
         self.assertEquals(versions['LATEST_THUNDERBIRD_DEVEL_VERSION'], "24.0b2")
         self.assertTrue("LATEST_THUNDERBIRD_ALPHA_VERSION" in versions)
         self.assertTrue(config.LATEST_THUNDERBIRD_ALPHA_VERSION.endswith('a2'))
-        self.assertEquals(len(versions), 3)
+        self.assertTrue("LATEST_THUNDERBIRD_NIGHTLY_VERSION" in versions)
+        self.assertTrue(config.LATEST_THUNDERBIRD_NIGHTLY_VERSION.endswith('a1'))
```

Comment:
You may want to copy the assert from Firefox which is more robust:
https://github.com/mozilla-releng/ship-it/blob/master/kickoff/test/views/test_json.py#L163-L177